Packs/CortexAttackSurfaceManagement/Scripts/GenerateASMReport/GenerateASMReport.yml

```diff
@@ -37,11 +37,14 @@ args:
 - description: Related or duplicate objects. (More of a nice to have because not sure how to track this).
   isArray: true
   name: asm_related
+- description: Matched remediation path rule (if there is a match).
```

Comment:
Should this be:
A comma-separated list of remediation path rules to match.

Reply:
this is an array